tests/custom_functions.cpp

```diff
@@ -0,0 +1,49 @@
+#include "lua_test.hpp"
+
+class IntWrapper : public luabind::Object {
+public:
+    IntWrapper(int value)
+        : _value(value) {}
+
+    int get() const {
+        return _value;
+    }
+
+public:
+    static int toLuaTable(lua_State* state) {
```

Comment:
Lets follow the overall style of the project and name the lua_State* variable `L`

Reply:
Done.